tests/test_pick.py

```diff
@@ -48,10 +48,47 @@ def test_multi_select():
 
 
 def test_option():
-    options = [Option("option1", 101), Option("option2", 102), Option("option3", 103)]
+    options = [Option("option1", 101, "description1"), Option("option2", 102),
```

Comment:
There is too many duplicated codes for this test case, maybe we can use some way to reduce them, like check the value in a for loop.